run_dir/design/base.html

```diff
@@ -15,7 +15,7 @@
 <script src="/static/js/moment.min.js"></script>
 
 <!-- Le styles -->
-<link type="text/css" rel="stylesheet" href="/static/nvd3/src/nv.d3.css">
+<!-- <link type="text/css" rel="stylesheet" href="/static/nvd3/src/nv.d3.css"> -->
```

Comment:
You are correct. This is a relic from when every page loaded every library. I changed this so that only the pages that needed stuff like D3 loaded it, but I must have missed this css import.

So yeah, remove from here but needs to be added into the template files for import anything like `<script src="/static/nvd3/*"></script>` (eg. the reads_vs_qv page)